lib/session.js

```
@@ -368,39 +368,44 @@ function sendLongSms(smsOptions, cb) {
+	//      be found from below reference.
+	//      reference: https://en.wikipedia.org/wiki/Concatenated_SMS
+	var udhHeaderSize = pduObj.params.short_message[0], // First octet is the size of UDH Header
+	    headerSize    = pduObj.params.short_message[2], // Header size other than first 2 octets
+	    csmsReference = pduObj.params.short_message.slice(3, 3 + headerSize - 2), // CSMS Reference starts from
+	                                                                              // 4th octet and length is
+	                                                                              // header size -2 octets
+	    partsCount    = pduObj.params.short_message[2 + csmsReference.length + 1],
+	    partNr        = pduObj.params.short_message[2 + csmsReference.length + 2],
+	    longSmsId     = pduObj.params.source_addr + '_' + pduObj.params.destination_addr + '_' + csmsReference;
+
+	        var msgId = require('uuid').v4();
```

ESLint

[error] Combine this with the previous 'var' statement.

(one-var)